modules/ray_service.py

```
@@ -469,8 +469,41 @@ def deploy_ray_service(
+
+    if deployment is None:  # pragma: no cover - deployment not registered
+        raise RuntimeError("LLMServeDeployment is not registered")
+
+    payload = {
+        str(key): str(value) for key, value in user_config.items() if value is not None
+    }
+
+    try:
+        deployment.update(user_config=payload)
+    except Exception as exc:  # pragma: no cover - Ray Serve API failure
+        raise RuntimeError("Failed to update Ray Serve deployment") from exc
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)